sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -101,7 +102,8 @@ class HFTransformersForecaster(BaseForecaster):
 
     def __init__(
         self,
-        model_path: str,
+        model_path: str = None,
```

Comment:
I would propose to use in future only model and set a deprecation warning for `model_path` because in my opinion this additional kwarg is not necessary, since model_path and model has basically the same meaning. 
@fkiraly any opinions

Reply:
@benHeid 
If that is the case, can we import the `warnings` dependency in the start?
Would it interfere in the checks during build process?